rpm/proxysql.spec

```diff
@@ -111,6 +111,7 @@ fi
 
 %post
 
+chown root:root %{_unitdir}/proxysql.service
```

Comment:
what about setting permissions during file installation?

Reply:
I have replaced the file permission attribute instead, which seemed like a better option.

Reply:
I've built the proxysql packages with this change and verified, the fix looks good:
```
[root@surabhi-ol8 percona]# ls -al /usr/lib/systemd/system/proxysql.service
-rw-r--r--. 1 root root 826 Aug 19 08:43 /usr/lib/systemd/system/proxysql.service
```